### DISCUSSION TITLE
When the user logins already...

### DISCUSSION THREAD

**Discussion by kezzayuno** [CMPUT404-Big-Yoshi-Social-Network/yoshi-connect#61] [General]
When the user logins in or signs up and then goes to the ```/welcome``` page, what should we do here? Same for if they try to go to ```/login``` or ```/signup```? Should we just redirect them (Status Code 301 maybe?)

**Comment by Tommy-Sand** [comment]
My thoughts are. When they go to the welcome page, we might want to just let them. However for login and sign up if they load these pages and we see that they are already logged in then we should redirect with a 301 to their feed. However we should keep some basic functionality wherein if a user sends a login request while they're already logged in we just update the expiry on their session token or something similar.
